tslib/src/functions-throttled/sync-results-to-sql/sync-results.sync.spec.ts

```diff
@@ -27,6 +27,7 @@ describe('SyncResultsService', () => {
     ;(mockSyncResultsDataService.prepareCheckResult as jest.Mock).mockReturnValueOnce(mockTransaction)
     ;(mockSyncResultsDataService.prepareAnswers as jest.Mock).mockReturnValueOnce(mockTransaction)
     ;(mockSyncResultsDataService.prepareEvents as jest.Mock).mockReturnValueOnce(mockTransaction)
+    ;(mockSyncResultsDataService.prepareDeviceData as jest.Mock).mockReturnValueOnce(mockTransaction)
```

Comment:
why the semi-colon prefix?

Reply:
It prevents the js engine from thinking the brackets are function call parameters to the preceding line.